google-cloud-debugger/google-cloud-debugger.gemspec

```diff
@@ -0,0 +1,39 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path("../lib/google/cloud/debugger/version", __FILE__)
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-debugger"
+  gem.version       = Google::Cloud::Debugger::VERSION
+
+  gem.authors       = ["Heng Xiong"]
+  gem.email         = ["hexiong@google.com"]
+  gem.description   = "google-cloud-debugger is the official library for Stackdriver Debugger."
+  gem.summary       = "API Client and instrumentation library for Stackdriver Debugger"
+  gem.homepage      = "http://googlecloudplatform.github.io/google-cloud-ruby/"
+  gem.license       = "Apache-2.0"
+
+  gem.files         = `git ls-files -- lib/* ext/*`.split("\n") +
+                      ["README.md", "LICENSE", ".yardopts"]
+  gem.require_paths = ["lib"]
+
+  gem.required_ruby_version = ">= 2.3.0"
```

Comment:
With help from Daniel, we got the gem to compile and work with Ruby 2.2!! However, the gem has too many low level dependencies on the language feature. We have little hope of getting it working with Ruby 2.1 and 2.0.

The good news is that the gem at least compiles and installs well with Ruby 2.1 and 2.0 (if I lift the version restriction in gemspec).